schema/metrics/histogram_metrics.mapping

```diff
@@ -0,0 +1,159 @@
+{
+  "index_patterns": [
+    "sso_metrics-*-*"
+  ],
+  "dynamic_templates": [
+    {
+      "attributes_map": {
+        "mapping": {
+          "type":"keyword"
+        },
+        "path_match":"attributes.*"
+      }
+    },
+    {
+      "resources_map": {
+        "mapping": {
+          "type": "keyword"
+        },
+        "path_match": "resources.*"
+      }
+    }
+  ],
+  "mappings": {
+    "_meta": {
+      "version": "0.1.0-dev"
+    },
+    "properties": {
+      "temporality": {
```

Comment:
This can only take one of the two values "AGGREGATION_TEMPORALITY_CUMULATIVE" or "AGGRGATION_TEMPORALITY_DELTA". Is it not possible to specify that here?

Reply:
Also, I think, it should be called "aggregationTemporaliity" instead of "temporality"

Reply:
yes- I've changed it accordingly 
and there is no way for limiting value in the mapping format - that is why I've added the json-schema file for enforcing exactly this...